RUNBOOK.md

```diff
@@ -0,0 +1,75 @@
+# Runbook for `paas-auditor`
+
+## How to deploy
+
+At the time of writing it is deployed manually rather than using our pipelines. We can integrate it into our pipelines if/when its data gets actually used for anything.
+
+The details below are GOV.UK PaaS-specific, but the general idea works for any Cloud Foundry.
+
+To run the below steps you'll need three things:
+
+* For your `cf` CLI to be logged into Cloud Foundry;
+* For the `DEPLOY_ENV` environment variable to be set appropriately;
+* For your AWS access to be configured so the `aws` CLI will work.
+* To have already created a Postgres service named `auditor-db`.
+
+```
+cd paas-auditor
+cf target -o admin -s billing
+cf push --no-start
+cf bind-service paas-auditor auditor-db
```

Comment:
Thoughts on this going in the manifest (separate PR)

Reply:
Go for it